test/OpenTelemetry.Exporter.Console.Tests/ConsoleActivityExporterTest.cs

```diff
@@ -0,0 +1,61 @@
+// <copyright file="ConsoleActivityExporterTest.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using OpenTelemetry.Tests;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace OpenTelemetry.Exporter.Console.Tests;
+
+public class ConsoleActivityExporterTest
+{
+    [Fact]
+    public void VerifyConsoleActivityExporterDoesntFailWithoutActivityLinkTags()
+    {
+        var activitySourceName = Utils.GetCurrentMethodName();
+        using var activitySource = new ActivitySource(activitySourceName);
+
+        var exportedItems = new List<Activity>();
+
+        using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+            .AddSource(activitySourceName)
+            .AddInMemoryExporter(exportedItems)
```

Comment:
This could be simplified by just using the extension method `AddConsoleExporter`. We don't have to test if activities are actually created in this test.

Reply:
The purpose of the InMemoryExporter was to inspect the Activity that was created and verify that it doesn't have Tags. This is the specific condition that was failing.

Reply:
Ahh I remember this from our discussion now. It's caught in the SimpleExportProcessor.